view/frontend/web/js/view/payment/method-renderer/afterpay.js

```diff
@@ -42,7 +43,8 @@ define(
         ko,
         checkoutData,
         selectPaymentMethodAction,
-        checkoutCommon
+        checkoutCommon,
+        dp
```

Comment:
"dp" it is too short name that is not self-explanatory. I suggest to modify into "datePicker" or something similar